holoviews/plotting/bokeh/element.py

```diff
@@ -370,9 +374,11 @@ def _init_plot(self, key, element, plots, ranges=None):
             properties['toolbar_location'] = self.toolbar
 
         properties['webgl'] = Store.renderers[self.renderer.backend].webgl
-        return bokeh.plotting.Figure(x_axis_type=x_axis_type,
-                                     y_axis_type=y_axis_type, title=title,
-                                     **properties)
+        with warnings.catch_warnings():
```

Comment:
Probably worth mentioning what warning you are catching and why it doesn't matter that you are suppressing it.